### DISCUSSION TITLE
How do I show both a CollectionView Header and EmptyView at the same time?

### DISCUSSION THREAD

**Discussion by kklose23** [dotnet/maui#16545] [General] [answered]
When a CollectionView ItemsSource is empty, it shows the EmptyView and also hides the Header.  How would I go about showing both?

**Comment by hartez** [comment, marked as answer]
Are you seeing this on Android? We recently fixed a bug on Android so that the Header and Footer would display at the same time as the EmptyView: https://github.com/dotnet/maui/pull/15979
